app/views/users/_search_form.html.erb

```diff
@@ -0,0 +1,4 @@
+  <%= bootstrap_form_tag url: '/users/search', remote: true, layout: :inline do |p| %>
```

Comment:
This is in part my own weird junk but I vastly prefer the the route syntax here. Can you swap this to use `users_search_path` instead of `url`?